resources/todomvc/vanilla-examples/javascript-es5/dist/view.js

```diff
@@ -49,9 +47,7 @@
     View.prototype._elementComplete = function (id, completed) {
         var listItem = qs(`[data-id="${id}"]`);
 
-        if (!listItem) {
-            return;
-        }
+        if (!listItem) return;
```

Comment:
return should be on its own line.

Reply:
linter did this automatically???

Reply:
This happens when I saved, so my editor called eslint and prettier automatically, but I'm not sure in which order. I'm also confused that some `dist` files changed even though the `src` didn't change, I only ran the build script for those...

Reply:
> I'm also confused that some `dist` files changed even though the `src` didn't change, I only ran the build script for those...

OK, that I now understand: the `src` files had already been changed according to the style before but the dist files hadn't been regenerated.

Reply:
> This happens when I saved, so my editor called eslint and prettier automatically, but I'm not sure in which order.

Recently Thorsten added a new plugin for eslint, and I hadn't run `npm ci` since then, so eslint was broken and nvim wouldn't tell me. So only prettier was run as a result. Now I fixed it and I don't have all these changes again.